generators/service-watson-conversation/templates/swift/instrumentation.swift

```diff
@@ -1,11 +1,11 @@
 import Foundation
 import LoggerAPI
+import CloudEnvironment
 import ConversationV1
 
-var conversation: Conversation!
-
-func initializeServiceWatsonConversation() throws {
+func initializeServiceWatsonConversation(cloudEnv: CloudEnv) throws -> Conversation {
     // Load credentials for Watson conversation using CloudEnvironment
+    let conversation: Conversation
```

Comment:
If you move this declaration below the guard you won't need the type annotation since it would be inferred by the assignment. You can also get rid of `conversationLocal` now there is no module-level var.

Reply:
done